packages/prettier-config/prettier-config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   singleQuote: true,
   printWidth: 120,
+  endOfLine: 'auto',
```

Comment:
Why is this needed to run tests on windows?

Reply:
@amphro without it linting fails because windows uses a different EOL character than linux and mac: https://eslint.org/docs/rules/linebreak-style